promql/promqltest/testdata/literals.test

```diff
@@ -57,3 +57,18 @@ eval instant at 50m 0 / 0
 
 eval instant at 50m 1 % 0
 	NaN
+
+eval instant at 50m ("Foo")
+	expect string `Foo`
+
+eval instant at 50m "Foo"
+	expect string "Foo"
+
+eval instant at 50m " Foo "
+    expect string " Foo "
+
+eval instant at 50m ("")
+	expect string
+
+eval instant at 50m ""
+	expect string
```

Comment:
Is this valid? I would expect this to fail now because there's nothing after `expect string`.